trymito.io/components/TextButton/TextButton.module.css

```diff
@@ -22,21 +19,47 @@
   display: inline-block;
 }
 
-.text_button:hover {
+.purple {
```

Comment:
I think we want to ideally not hardcode colors anywhere, if possible. Can we call this something similar to --color-text-primary (e.g. .primary-with-highlight-background or something). Same for the rest of these.